microsoft-edge/devtools-guide-chromium/storage/cookies.md

```diff
@@ -30,16 +30,15 @@ See [Using HTTP cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies).
 <!-- ====================================================================== -->
 ## Open the Cookies pane
 
-1. [Open DevTools](/microsoft-edge/devtools-guide-chromium/open).
+1. Open DevTools on the webpage you want to edit. To open DevTools, use one of the techniques described in [Open DevTools](/microsoft-edge/devtools-guide-chromium/open).
 
-1. Click the **Application** tab to open the **Application** panel.  The **Manifest** pane opens:
+2. Click the **Application** icon (![Application icon in Activity Bar](images/application-storage-cookies-application-icon.png)) in the Activity Bar or select "Application" in the Quick View dropdown list to open the **Application** panel.  The **Manifest** pane opens:
 
-   ![The Manifest pane.](../media/storage-application-manifest-empty.msft.png)
+   ![The Manifest pane.](images/application-storage-cookies-application-pane.png)
 
-1. Under **Storage**, expand **Cookies**, then select an origin:
-
-   ![The Cookies pane.](../media/storage-application-storage-cookies-selected.msft.png)
+3. Under **Storage**, expand **Cookies**, then select an origin:
 
+   ![The Cookies pane.](images/application-storage-cookies-expand-cookies-select-origin.png)
```

Comment:
```suggestion
   ![The Cookies pane.](cookies-images/application-storage-cookies-expand-cookies-select-origin.png)
```
moving to dedicated image dirs

Reply:
Nice! Although I can see one problem with doing this now: it will make it harder to merge the incoming changes from the `focus-mode-docs-bash` branch. We have tons of image changes coming in that branch, and we're not quite ready to merge them yet. So, we have to maintain the branch in parallel to main for a while still. Therefore, any changes to the path of images in main risk introducing merge conflicts in the focus-mode-docs-bash.
I would prefer if we didn't change any image paths until we merge the focus mode branch. And once that's done, we can move all images automatically in one go, with a script.

Reply:
(This is an "Outdated" comment thread.)  @captainbrosset - it's your call, I suggest keeping the original filenames, for tracking.  @dan-wesley

Reply:
Dan (or Capt.) pls close this "Outdated" thread if feasible.

Reply:
Yes, I would prefer it if we could keep the same image names and paths if possible. This way when merging the `focus-mode-docs-bash` branch, it will just be a question of choosing which image is best.

Reply:
@dan-wesley (or @captainbrosset )  Please close this "Outdated" conversation as Resolved if this has been resolved in the latest files.